app/src/androidTest/java/org/kiwix/kiwixmobile/utils/files/FileUtilsInstrumentationTest.java

```diff
@@ -27,7 +27,7 @@
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
-import org.kiwix.kiwixmobile.core.entity.LibraryNetworkEntity.Book;
+import org.kiwix.kiwixmobile.core.entity.LibraryNetworkEntity;
```

Comment:
Here also you can import directly  this `import org.kiwix.kiwixmobile.core.entity.LibraryNetworkEntity.Book;`